aggregators/credbull/index.ts

```diff
@@ -0,0 +1,28 @@
+import { FetchOptions, FetchResultVolume, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+
+const CBL_TGE = 1730973600; // 2024-11-07 10:00 UTC
+const CBL_TOKEN = "0xD6b3d81868770083307840F513A3491960b95cb6";
+const CBL_TRANSFER_EVENT_ABI = "event Transfer(address indexed from, address indexed to, uint256 value)";
+
+async function fetch(options: FetchOptions): Promise<FetchResultVolume> {
+  const dailyVolume = options.createBalances();
+  const logs = await options.getLogs({
+    target: CBL_TOKEN,
+    eventAbi: CBL_TRANSFER_EVENT_ABI,
```

Comment:
why are you treating  all CBL token transfers as your app volume?

Reply:
* I'm pretty n00b, so 'all transfers' seems like the volume of the token to me. What is a better definition? Include mint/burn? I'll look further.
* I wanted to get some expert feedback on review, to make sure I was on the right track. 
* The docs are sparse, so I was trying to figure out the means of interrogating the chain, via `getLogs`.
* I haven't checked the other methods (`sdk.graph` etc.)

That said, after team discussions, I will change this to focus on our staking vault, not token. Will close PR, as not correct at all.